packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -266,182 +266,182 @@
     "callingNetworkFailure": {
       "title": "Troubling connecting call",
       "message": "You seem to be offline. Please check your network connection and try again.",
-      "closeButtonAriaLabel": "Close"
+      "dismissButtonAriaLabel": "Close"
```

Comment:
Should the content be different? or are we okay with it still stating as the "close button"?

Reply:
Figma deems this to be close. So I think this should remain.
Dismiss is more so for Contoso understanding.